### PR TITLE
chore(ngInject): Use ng-annotate-loader for angular DI annotation

### PR DESCRIPTION
**Pull request by christopherthielen**

### CODE REVIEW

File: app/scripts/modules/core/api/api.service.ts [hunk 1]
```diff
@@ -49,7 +46,7 @@ export class Api {
         const isZeroLengthHtml = (contentType.includes('text/html') && (result.data === ''));
         const isZeroLengthText = (contentType.includes('text/plain') && (result.data === ''));
         if (!(isJson || isZeroLengthHtml || isZeroLengthText)) {
-          this.authenticationIntializer.reauthenticateUser();
+          this.authenticationInitializer.reauthenticateUser();
```
Comment (jrsquared) [inline app/scripts/modules/core/api/api.service.ts]: Ha! It fixes spelling mistakes too!

**Comment by christopherthielen** [comment]
I'm fairly confident I caught all the renames

**Comment by christopherthielen** [comment]
@danielpeach PTAL, we're moving to `ng-annotate-loader`

**Comment by anotherchrisberry** [comment]
Let's find out!

**Comment by danielpeach** [comment]
LGTM - thanks.

**Pull request merged**